kpi/serializers/v2/asset.py

```diff
@@ -135,7 +136,10 @@ def update(self, asset, validated_data):
             except ValueError as err:
                 raise serializers.ValidationError(str(err))
             validated_data['content'] = asset_content
-        return super().update(asset, validated_data)
+        try:
+            return super().update(asset, validated_data)
+        except ValidationError as e:
+            raise serializers.ValidationError(e.message)
```

Comment:
I think the accepted thing to do these days would be something like `raise serializers.ValidationError(e.message) from e`.  Here's the [documentation](https://docs.python.org/3/library/exceptions.html#built-in-exceptions).

Reply:
Thanks for this.
I've made that change